From 8ee50c8ba339ded0ad127420dc299be409d438ed Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 5 Oct 2022 14:28:47 +0200
Subject: [PATCH 019/529] ARM: configs: add CONFIG_SCHED_MC in multiv7_addons
 fragment

Multi-Core scheduler improves the CPU scheduler's decision making when
dealing with multi-core CPU chips. There are 2 cores on STM32MP157, so
enable CONFIG_SCHED_MC.

Change-Id: I140cb8e0ff77244adc2383a67f1a604bf2a776f2
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -27,6 +27,7 @@ CONFIG_POWER_RESET_SYSCON=y
 CONFIG_PREEMPT=y
 CONFIG_PREEMPT_COUNT=y
 CONFIG_PROFILING=y
+CONFIG_SCHED_MC=y
 CONFIG_SENSORS_IIO_HWMON=y
 CONFIG_SERIAL_NONSTANDARD=y
 CONFIG_SRAM=y
